.github/workflows/ci.yml

```diff
@@ -30,6 +30,24 @@ jobs:
             - run: npm run lint:ts
 
     tests:
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: false
+            matrix:
+                contract-name: ["Safe", "SafeL2"]
+        env:
+            SAFE_CONTRACT_UNDER_TEST: ${{ matrix.contract-name }}
+        steps:
+            - uses: actions/checkout@v4
+            - uses: actions/setup-node@v4
+              with:
+                  node-version: ${{ env.NODE_VERSION }}
+                  cache: "npm"
```

Comment:
I don't think the matrix is needed here anymore, the `npm run test` runs all the necessary commands (l1, l2, zk, etc)

Reply:
Updated 👍🏾